app/testOnly/controllers/ChargeBreakdownController.scala

```diff
@@ -61,7 +60,7 @@ class ChargeBreakdownController @Inject()(authorisedController: AuthorisedContro
     implicit user => DDInterrupt.interruptCheck { _ =>
       if(appConfig.features.interestBreakdownEnabled()) {
         serviceInfoService.getPartial.map { navLinks =>
-          InterestChargeViewModel.form.bindFromRequest.fold(
+          EstimatedInterestViewModel.form.bindFromRequest.fold(
```

Comment:
Could you please also rename the above controller action, and the associated route name (placeholder for now). I did the task recently to add the new action and I called it `crystallisedInterestBreakdown`, with the route being `/crystallised-interest` so I think it makes sense to follow that pattern here too